database/DevHome.Database/DatabaseModels/RepositoryManagement/RepositoryMetadata.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace DevHome.Database.DatabaseModels.RepositoryManagement;
+
+/// <summary>
+/// This represents the SQLite model EF has in the database.
+/// Any change here needs a corresponding migration.
+/// Use FluentAPI in the database context to further customize each column.
+/// </summary>
+public class RepositoryMetadata
```

Comment:
No index for this?

Reply:
Technically I believe no index is needed because the C# models guarantee the 1:1 relationship.

I am considering removing `RepositoryMetadata` and putting `IsHidden` inside `Repository`

I'm leaving this as is for this PR.  If I do keep the 1:1 I'll look into making an index.